frontend/src/views/Game.tsx

```diff
@@ -275,15 +296,15 @@ function GamePage() {
 
           // If this user refreshed and has already submitted code, load and save their latest code
           res.players.forEach((player) => {
-            if (player.user.userId === location.state.currentUser.userId && player.code) {
-              setDefaultCodeFromProblems(res.problems, player.code, [player.language as Language]);
+            if (player.user.userId === location.state.currentUser.userId) {
+              setDefaultCodeFromProblems(res.problems, player.submissions);
```

Comment:
Can you explain the change that was made here?

Reply:
So, whenever the page was revisited by someone who left, this callback is called. Previously, when this happened, setDefaultCodeFromProblems() took a field for code and language, as their was only one problem to save for. However, since there are multiple problems, the method has been changed to take the submissions from which the code is now taken.